boards/arm/nucleo_wl55jc/board.cmake

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
may be worth adding stm32cubeprogrammer support

Reply:
stm32cubeprogrammer added